.github/workflows/docker.yml

```
@@ -88,7 +88,7 @@ jobs:
       name: build
       run: |
         # rebuild sirf image
-        ./docker/compose.sh -bR${{ matrix.type == 'gpu' && 'g' || 'c' }}
+        ./docker/compose.sh -bR${{ matrix.type == 'gpu' && 'g' || 'c' }} -- --build-arg BUILD_CIL=ON
```

if we only want BUILD_CIL for `edge` but not `latest,M.m.p` then...

```suggestion
        ./docker/compose.sh -bR${{ matrix.type == 'gpu' && 'g' || 'c' }} ${{ github.ref == 'refs/heads/master' && '-- --build-arg BUILD_CIL=ON' || '' }}
```